docs/administration-guide/databases/impala.md

```diff
@@ -0,0 +1,139 @@
+## Working with Impala in Metabase
+
+Metabase provides a driver for connecting to Impala databases. Under the hood, Metabase uses Impala's JDBC driver; due to licensing restrictions, we can't
```

Comment:
What license is the driver released under? I can't find a reference at the DL page below.